frontend/components/MessagingModal.tsx

```
@@ -137,206 +149,225 @@ export default function MessagingModal({ isOpen, onClose, currentUser }: Messagi
-              Include translated version of this message
-            </Label>
+            {/* Translation Option */}
+            <div className="flex items-center space-x-2 p-3 bg-gray-50/50 rounded-xl">
+              <button
+                onClick={() => setIncludeTranslation(!includeTranslation)}
+                className={`h-5 w-5 rounded-full border-2 flex items-center justify-center transition-all ${
+                  includeTranslation 
+                    ? 'bg-gradient-to-br from-green-500 to-green-600 border-green-600 text-white shadow-sm' 
+                    : 'border-gray-300 hover:border-gray-400'
+                }`}
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)